docs/getting-started.md

```diff
@@ -30,13 +42,6 @@ This is a summary of setup steps needed to install and work with React Native fo
 - Install [Chrome](https://www.google.com/chrome/) (_optional_, but needed for JS debugging)
 - Install [Yarn](https://yarnpkg.com/en/docs/install) (_optional_ if consuming react-native-windows, but required to work in the react-native-windows repo)
 
-## Installing React Native for Windows (vnext)
-
-There are two ways to install and work with React Native for Windows (vnext):
-
-1. [Consuming react-native-windows](consuming-rnw.md) : Using react-native-windows in your package
-1. [Building react-native-windows repo](building-rnw.md) : Manually by cloning the repo and get things running.
-
 ## E2E Test

```

Comment:
I think this can be removed from here. Consider the primary audience is consumers of react native windows, app developers. Having a separate section for those who contribute with E2E test docs as part of that would be better.

Reply:
I did remove this - So I agree!